apps/webviews/src/styles/main.css

```
@@ -1,5 +1,9 @@
 @import 'tailwindcss';
 
+/* The theme builder (rendered in the theme-builder webview) lives in its own
+   workspace package, so its sources must be scanned explicitly for utilities. */
+@source '../../../theme-builder/src';
```

Stylelint (17.13.0)

[error] Unexpected unknown at-rule "`@source`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)